src/jira/client/axios.ts

```diff
@@ -149,13 +149,13 @@ const instrumentRequest = (response) => {
  * @param logger
  */
 const instrumentFailedRequest = (instance: AxiosInstance, logger: Logger) => {
-	return async (error) => {
+	return async (error: AxiosError) => {
 		instrumentRequest(error?.response);
-		if (error?.statusCode === 503) {
+		if (error.response?.status === 503) {
 			const statusResponse: AxiosResponse = await instance.get("/status");
```

Comment:
I'm pretty sure that  doing an await on that request will throw an error if it returns a 503.  You need to either catch it or do a try/catch around it